contrib/docker/Dockerfile

```diff
@@ -1,60 +1,50 @@
 FROM python:3.6
 
-MAINTAINER Xiao Hanyu <hanyu.xiao@shopeemobile.com>
-
-# Add a normal user
-RUN useradd --user-group --create-home --shell /bin/bash work
-
```

Comment:
I've read a lot docs about docker practice and it seems not good to run programs inside docker as root, so I create a new `work` user.

However, I think if we focus these docker files on development, then it's OK.

Reply:
Oh, I didn't know, if you have some documentation on that I would be interested.

I removed it because it's a pain to manage a user stuffs in docker :)

Reply:
https://medium.com/@mccode/processes-in-containers-should-not-run-as-root-2feae3f0df3b

See some articles like this.

However, I think for development purpose, it's OK to simplify things.

Reply:
@kakoni if we do that, the advantage is that we could simply remove all the content with superset from the image and just provide the dev environment needed for it… but in that case, wouldn't it be easier to just something like virtualenv? ;)

Reply:
I think we can provide two docker images? one for development and one for production?